recon/src/client.rs

```diff
@@ -98,14 +98,6 @@ where
         rx.await?
     }
 
-    /// Report all keys in the range that are missing a value
-    pub async fn keys_with_missing_values(&self, range: RangeOpen<K>) -> Result<Vec<K>> {
-        let (ret, rx) = oneshot::channel();
-        self.sender
-            .send(Request::KeysWithMissingValues { range, ret })
-            .await?;
-        rx.await?
-    }
     /// Report the local nodes interests.
     pub async fn interests(&self) -> Result<Vec<RangeOpen<K>>> {
```

Comment:
Do we want to keep `RangeOpen` around? It seems like we're just turning it into a normal range now but it seems to be working fine.

Reply:
Yes, RangeOpen is still useful for communicating interests. Its another way of saying interests boundaries are always fenceposts.